src/config.js

```diff
@@ -8,13 +8,10 @@ activityEvents.push("keydown");
 export const config = {
   app: {
     activityEvents,
-    apiKey:
-      process.env.REACT_APP_API_KEY ||
-      "PXPWlQGB3wKXotkWN1PbSwbSoM7CoWW0ZMPWYtfc",
-    googleAnalyticsEnabled:
-      process.env.REACT_APP_GOOGLE_ANALYTICS_ENABLED || "false",
-    googleAnalyticsContainerId:
-      process.env.REACT_APP_GOOGLE_ANALYTICS_CONTAINER_ID || "",
+    apiKey: process.env.REACT_APP_ECMPS_API_KEY,
+    googleAnalyticsEnabled: process.env.REACT_APP_GOOGLE_ANALYTICS_ENABLED || 'false',
+    googleAnalyticsPublicContainerId: process.env.REACT_APP_GOOGLE_ANALYTICS_PUBLIC_CONTAINER_ID || 'GTM-TT724Q9',
```

Comment:
I would take the keys out of the app and default to blanks.  as a rule, keys are not included in source